bin/node/cli/benches/block_production.rs

```diff
@@ -0,0 +1,284 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
```

Comment:
Maybe a small doc on what this benchmark does, and what assumptions it makes?

Reply:
Hmm.... well, I could certainly write something out, but considering how relatively simple the benchmark is (and the fact that I'm not yet super familiar with all of the internals) I'm not entirely sure *what* extra information I could add with such a comment. (: